src/components/admin/navigation/AdminLayout.tsx

```diff
@@ -91,7 +91,6 @@ const closedMixin = (theme: Theme): CSSObject => ({
   },
 })
 
-// @ts-expect-error bad MUI types for styled.div
```

Comment:
@stilian-nikolaev Why you've removed this line, indeed there is a misalignment of the types so that's why this exists